dev-haskell/http-media/http-media-0.8.0.0.ebuild

```diff
@@ -23,10 +23,10 @@ RDEPEND=">=dev-haskell/case-insensitive-1.0:=[profile?] <dev-haskell/case-insens
 "
 DEPEND="${RDEPEND}
 	>=dev-haskell/cabal-1.22.2.0
-	test? ( >=dev-haskell/quickcheck-2.8 <dev-haskell/quickcheck-2.14
-		>=dev-haskell/test-framework-0.8 <dev-haskell/test-framework-0.9
-		>=dev-haskell/test-framework-quickcheck2-0.3 <dev-haskell/test-framework-quickcheck2-0.4 )
 "
+	# test? ( >=dev-haskell/quickcheck-2.8 <dev-haskell/quickcheck-2.14
```

Comment:
I'm not sure this commit does what it says it does.

Reply:
It didn't, this was poor use of `git commit -a -C ORIG_HEAD` on my part. this should be fixed now (after making the same mistake again, and fixing that....)